content/workflows/error-handling.md

```diff
@@ -18,14 +19,38 @@ Two types of errors can occur during task execution: *Recoverable* and *NonRecov
 
 If a *NonRecoverable* error occurs, the workflow execution will fail, similarly to the way described for when an error is raised from the workflow itself.
 
-If a *Recoverable* error occurs, the task execution might be attempted again from its start. This depends on the configuration of the `task_retries` parameter, which determines how many retry attempts will be given by default to any failed task execution.
+If a *Recoverable* error occurs, the task execution might be attempted again from its start. This depends on the configuration of the `task_retries` and `max_retries` parameters, which determines how many retry attempts will be given by default to any failed task execution.
 
-The `task_retries` parameter can be set in one of the following manners:
+The `task_retries` and `max_retries` parameters can be set in one of the following manners:
 
-* When bootstrapping using the [Cloudify CLI](manager-blueprints-bootstrapping.html), the `task_retries` parameter is a configuration parameter under `cloudify`.`workflows`.
+* If the operation [`max_retries` parameter](blueprints-spec-interfaces.html#definition) has been set for a certain operation, it will be used.
 
-* If Cloudify wasn't bootstrapped using Cloudify's CLI, the `task_retries` parameter may be set via a REST call to the management server that creates a provider context object, but this call is not yet documented.
+* When [bootstrapping](manager-bootstrapping.html), the manager blueprint `task_retries` parameter is a configuration parameter under the `manager_configuration` node template under the `cloudify`.`workflows` property.
+
+* If Cloudify wasn't bootstrapped using Cloudify's CLI, the `task_retries` parameter may be set via a REST call to the management server that creates a provider context object.
 
 If the parameter was not set, it will default to the value of `-1`, which means *infinite retries*.
 
-In addition to the `task_retries` parameter, there's also the `retry_interval` parameter, which determines the minimum amount of wait time (in seconds) after a task execution fails before it is retried. It can be set in the very same way `task_retries` is set. If it isn't set, it will default to the value of `30`.
+In addition to the `task_retries` parameter, there's also the `retry_interval` parameter, which determines the minimum amount of wait time (in seconds) after a task execution fails before it is retried. It can be set in the very same way `task_retries` and `max_retries` are set. If it isn't set, it will default to the value of `30`.
+
+# Lifecycle Retries (Experimental)
```

Comment:
as a reader, i think i'd wonder why exactly the parameter is called "subgraph_retries", rather than "node_retries" or "lifecycle_retries"